disperser/batcher/encoding_streamer.go

```diff
@@ -137,7 +137,11 @@ func (e *EncodingStreamer) Start(ctx context.Context) error {
 						// ignore canceled errors because canceled encoding requests are normal
 						continue
 					}
-					e.logger.Error("error processing encoded blobs", "err", err)
+					if strings.Contains(err.Error(), "too many requests") {
+						e.logger.Warn("error processing encoded blobs", "err", err)
```

Comment:
downgraded this case as it happens quite often and it's safe to ignore

Reply:
Maybe remove `error` from the Warning message. So what is the source of `too many requests`? Are we being rate limited when this occurs?